.circleci/test_runner.py

```diff
@@ -17,11 +18,8 @@ def create_job(test_runner: str) -> str:
             "cwd": "/home/omg/omg-js"
         }
     }
-    try:
-        request = requests.post(test_runner + '/job', json=payload)
-    except ConnectionError:
-        logging.critical('Could not connect to the test runner')
-        sys.exit(1)  # Return a non-zero exit code so CircleCI fails
+
+    request = requests.post(test_runner + '/job', json=payload)
```

Comment:
Why remove the try/except block?

Reply:
Each time there is a `try/catch` block it logs it and `sys.exit(1)`. There was a lot of code duplication for the posting to slack logic for each exception. 

And since the script is just going to exit and not execute anything anymore after each `try/catch`, it made more sense to just raise an exception there (with the relevant error code), and then have a `try/catch` around `start_workflow` (one of the main entrypoints), and handle the logic for exception handling and posting to slack from there - less code to maintain but still has the same logic

Reply:
With the `try/except` block you get the ability to add a log message so that any user checking the CircleCI logs can see why it failed.